tests/test_views.py

```diff
@@ -250,7 +250,7 @@ def test_update(bukudb, client, override):
 @pytest.mark.slow
 @pytest.mark.parametrize('redirect, uri, args', [
     ('_add_another', '/bookmark/new/', {'url': '/bookmark/'}),
-    ('_continue_editing', '/bookmark/edit/', {'id': '1'}),
+    ('_continue_editing', '/bookmark/edit/', {'id': '1', 'url': '/bookmark/'}),
```

Comment:
This parameter not being passed [was a bug in Flask-Admin](https://github.com/pallets-eco/flask-admin/issues/2311)